### PR TITLE
Fixes for DDS Examples

### PR DESCRIPTION
**Pull request by EduPonz**
This PR fixes some installation issues for the DDS Examples

### CODE REVIEW

File: examples/C++/DDS/Configurability/README.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 --------------------------------------------------------
-- Use Case Publisher-Subscriber for eProsima Fast DDS -
+- ConfigurabilityPublisher Publisher-Subscriber for eProsima Fast DDS -
```
Comment (JLBuenoLopez-eProsima) [inline examples/C++/DDS/Configurability/README.txt:2]:
```suggestion
- Configurability Publisher-Subscriber for eProsima Fast DDS -
```

**Review by JLBuenoLopez-eProsima — COMMENTED** [review]
LGTM. Only a small nitpick

**Comment by EduPonz** [comment]
I've added the `skip-ci` label since the changes only affect examples

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Pull request merged**